node-agent/workload_manager/src/workload_manager/workload/errors.rs

```diff
@@ -0,0 +1,27 @@
+use std::error::Error;
+use std::fmt;
+
+#[derive(Debug)]
+pub struct WorkloadError {
+    details: String,
+}
+
```

Comment:
You should use the `anyhow` crate instead.

Reply:
Sorry, I don't understand. 
Does this mean that I'm supposed to change the #[derive(Debug)] for something from anyhow crate (but I didn't see anything that looks like it in the doc) or should I just remove WorkloadError struct and just use anyhow basics errors instead ?

Reply:
`WorkloadError` is only a string, the `anyhow` create already define an error type that does exactly this, implements `Error`, etc. So you could remove `WorkloadError` and simply use `anyhow` instead.

Reply:
Ok, thank you, that's a lot more practical indeed !